### DISCUSSION TITLE
can't convert OCRv4 to nb with paddlelite v2.10

### DISCUSSION THREAD

**Discussion by balcklive** [PaddlePaddle/PaddleOCR#12802] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：ubuntu 22.04
- 版本号/Version：Paddle：2.5.1  PaddleOCR：release/2.7  问题相关组件/Related components：paddle lite 2.10
- 运行指令/Command Code：
./opt_linux --model_file=inference.pdmodel --param_file=inference.pdiparams       --optimize_out_type=naive_buffer       --optimize_out=quant_opt       --valid_targets=arm
- 完整报错/Complete Error Message：
-Loading topology data from inference.pdmodel
Loading params data from inference.pdiparams
1. Model is successfully loaded!
[F  8/27  2:35:29.277 ...-Lite/lite/core/model/general/op_desc.cc:63 Output] Check failed: it != outputs_.end(): 
已放弃 (核心已转储) 

备注：尝试了使用paddlelite2.12版本执行上述命令是可以正常运行的

在https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/models_list.md上分别下载了推理模型和训练模型，使用推理模型直接运行上述代码是可以转换的，但是使用训练模型先执行
python tools/export_model.py -c configs/rec/PP-OCRv4/en_PP-OCRv4_rec.yml -o Global.pretrained_model=E:\02-ocr\03-infer_models\trained_own\best_accuracy Global.save_inference_dir=E:\02-ocr\03-infer_models\trained_own
然后再执行上述命令时，则出现上述报错

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by DennisJcy** [comment]
遇到了同样的问题，预训练模型训练后无法opt。推理模型下载下来可以直接opt。请问有解决吗 @balcklive

**Comment by balcklive** [comment]
> 遇到了同样的问题，预训练模型训练后无法opt。推理模型下载下来可以直接opt。请问有解决吗 @balcklive

没有，只能使用Paddlelite 2.12才行
